app/index.md

```diff
@@ -22,4 +22,8 @@ Welcome to _Flavor_, the only place on the planet where your taste buds won't be
 
 ### [Mac Gump](writer/mac-gump.md) | macgump@flavor.magazine
 
-[Caprese Mac and Cheese](recipe/feb/caprese-mac-and-cheese.md)
\ No newline at end of file
+[Caprese Mac and Cheese](recipe/feb/caprese-mac-and-cheese.md)
+
+### [Vegetarian Chili](writer/mac-gump.md) | michele-oliviero@flavor.magazine
```

Comment:
After writer it was necessary to indicate your new file  replacing mac-gump.md with vegetarian-chili-mar.md